pkg/ccl/backupccl/testdata/backup-restore/import-start-time

```diff
@@ -0,0 +1,108 @@
+# Test that Import properly sets and removes the ImportStartTime from the table descriptor.
```

Comment:
This seems like it should be in pkg/sql/importer? 

Or is the logic that backupccl is the one that cares that importer is correctly persisting offline-as-of times in tables and so backupccl is where we test it, even though it is importer doing the persisting?

Reply:
Funny story. This _should_ be in importerccl, and if you allow it, I'll move it there during stability.

 I[ tried ](https://github.com/cockroachdb/cockroach/pull/85637)excavating the backupccl data driven framework into its own package so import could use it, but I'm currently stuck diagnosing some weird dependency cycle. I plan to figure this out during stability.